themes/default/content/blog/introducing-dev-releases/index.md

```diff
@@ -0,0 +1,69 @@
+
+<!--more-->
+
+## What are dev releases?
+
+On every push to the main branch, in other words every time a pull request is merged, we publish a new dev release for both the CLI and the Pulumi core SDKs.  For the CLI, we publish the binaries for all supported platforms to S3, while for the SDKs we create pre-release versions in whatever form the various package managers support.
+
+This means these dev releases always include all the latest features and bug fixes, that will be in the next stable release.  And while it was always possible to get the latest CLI or SDKs with some moderate effort depending on the platform and language, having these be proper releases makes for a much more standardized experience on how to install them.
+
+Dev SDKs are currently available for NodeJS, Python, Dotnet and Go.
+
+## How are we using them internally at Pulumi?
+
+At Pulumi we are not only developers of our product, but we also love using it internally.  All deploys for Pulumi Cloud are orchestrated by Pulumi.  In addition we use Pulumi for managing other things such as this blog, and even access management to services we are using is partly done with Pulumi.  All of these are a great way for us to test our software in real world use cases, before releasing it.
+
+In addition to that, we also use the dev releases for testing both our [templates](https://github.com/pulumi/templates/) and [examples](https://github.com/pulumi/examples/).  This is a huge set of Pulumi programs, that can help figure out problems early.
+
+## How you can leverage dev releases
+
+In addition to making it easier for us to test new code that gets merged, and making sure we can release the most stable software possible, we also wanted to make it easier for users to use these dev releases.  There's a few reasons why these could help your organization:
+
+- A bug fix has been merged, but not yet released?  You can get it a few minutes after it is merged via the dev release channel
+- You want to make sure your production stacks work well with the latest version of Pulumi?  Use dev releases with your dev stacks for some early testing.  We'd be delighted with any bugs that are reported before they make it into a release.
+
+## How to get started
+
+For instructions on how to use the CLI dev releases see the [Install Docs](https://www.pulumi.com/docs/install/ TODO: link to the appropriate header when https://github.com/pulumi/pulumi-hugo/pull/4097 is merged), and for pre-release versions of dev SDKs see the docs for the appropriate [language SDK](https://www.pulumi.com/docs/languages-sdks/).
```

Comment:
Would be nice to link direclty to the header for the appropraite section in each language SDK as well here once that other PR merges.